modules/exploits/linux/http/librenms_cmd_injection.rb

```diff
@@ -0,0 +1,160 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  include Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'LibreNMS Cmd Injection',
```

Comment:
Name the functionality, and ensure the file name matches.

Consider:

```suggestion
      'Name'           => 'LibreNMS addhost Command Injection',
```